### ISSUE TITLE
Can I train on cpu and control layer freezing ? 

### ISSUE THREAD

**Issue by Faisal-Hajari** [yumeng5/LOTClass#21]

**Comment by yumeng5** [comment]
Hi,

While it's possible to freeze most layers to save computation and make it possible to train on CPUs, it's not recommended to do so -- the hyperparameters (e.g., learning rate, batch size) will need to be re-tuned and there's no guarantee on how well the trained model will be. It's probably better to seek for GPU resources like Google Colab. 

Best,
Yu
